tests/typowalking/wordwalker.js

```diff
@@ -27,7 +28,8 @@ bender.test( {
 		wordwalker = new editor.plugins.nanospell.WordWalker(range);
 
 		while (currWordObj = wordwalker.getNextWord()) {
-			wordsReturned.push(currWordObj.word);
+			word = currWordObj.word;
+			if (word) wordsReturned.push(word);
```

Comment:
Can you think of any ways to represent the original bug as a test?  I think trying to time it will suck badly, but is there any way to maybe set the html and just trigger the specific walking event?